pkg/stanza/fileconsumer/file_windows.go

```diff
@@ -12,11 +12,10 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/reader"
 )
 
-func (m *Manager) preConsume(ctx context.Context, newReaders []*reader.Reader) {
-	return
+func (m *Manager) preConsume(_ context.Context, _ []*reader.Reader) {
 }
 
-// On windows, we close files immediately after reading becauase they cannot be moved while open.
+// On windows, we close files immediately after reading because they cannot be moved while open.
```

Comment:
I'm questioning the quality of the current misspell used during build: I spotted this typo while changing the code near by, this was not flagged by misspell.

Reply:
Strange. Good catch though.